README.md

```diff
@@ -43,13 +40,16 @@ app locally. Do this like so:
     cd yari
     yarn install
 
-Now run the following command to create a `.env` file inside your `yari` repo
-root and set the `CONTENT_ROOT` environment variable equal to the path to the
-`content` repo. This is so the Yari app can find the content it needs to render.
-You'll need to replace `/path/to/mdn/content/files` with the path to the
-`/files` folder inside your clone of the `content` repo:
+Now copy the `.env-dist` file to `.env`:
+
+    cp .env-dist .env
 
-    echo CONTENT_ROOT=/path/to/mdn/content/files >> .env
+If you followed the instructions above and cloned the `content` repo as a
+sibling of your `yari` repo, the `CONTENT_ROOT` environment variable is already
+set and Yari will be able to find the content it needs to render. If not, and
+you cloned the `content` repo elsewhere, you'll need to modify the `.env` file,
+replacing `../content/files` with the path to the `/files` folder inside your
+clone of the `content` repo.
```

Comment:
I think mentioning the  `/files` bit is important as it's and easy and _unnecessary_ mistake.

Reply:
As well as resolving relative paths, we also automatically correct that:

https://github.com/mdn/yari/blob/13b8c2f85cc4931ab259cd1922e1c558f4f43cf4/content/constants.js#L52-L56

Since it warns in console as well, do we need to explicitly call it out here?